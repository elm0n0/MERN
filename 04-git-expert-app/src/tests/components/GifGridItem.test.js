import React from 'react'
import '@testing-library/jest-dom';
import {shallow} from 'enzyme';
import GifGridItem from '../../Component/GifGridItem';

describe('GifGridItem Test', () => {

    const titulo = 'titulo';
    const url = 'https://miurl.es';

    test('mostrar el componente <GifGridItem /> ', () => {
        const wraper = shallow(<GifGridItem 
            title={titulo} 
            url={url}
        />);
        expect(wraper).toMatchSnapshot();
    })
})

