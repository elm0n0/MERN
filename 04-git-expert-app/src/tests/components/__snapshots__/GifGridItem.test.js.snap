// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem Test mostrar el componente <GifGridItem />  1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="titulo"
    src="https://miurl.es"
  />
  <p>
    titulo
  </p>
</div>
`;
